src/installer/Directory.Build.targets

```diff
@@ -82,55 +72,6 @@
       <UseShippedHostPolicyPackage>false</UseShippedHostPolicyPackage>
```

Comment:
For example, this is hardcoded to `false` (we can't control it from anywhere), so I was thinking maybe `HostPolicyVersion` which is falling back to 2.0.0 can be safely removed? Even if we want to keep `HostPolicyVersion` etc. for some reason, the hardcoded `false` values make these additional switch properties useless. Only way is to update this file manually, so we can leave line 73 unconditional and delete 71 and 74 (same for other similar props above).

Reply:
Thanks for the suggestion. I just took a look and I think the properties make sense the way they currently are (based on the descriptions above each block). Hopefully we can move these properties over to src\native\corehost\ after consolidating building and packaging of the host.

Reply:
Yup, also `UseShippedHostPolicyPackage` etc. are only used in this file. I think if we get some feedback whether we solely rely on git branching and always build with same versions, these individual version properties can be unified in eng/Versions.props. Then we won't be needing this target.